handlers.js

```diff
@@ -70,13 +74,36 @@ export function makeRows({tests, columns}) {
     row.cells[columnIndex] = current;
     return rows;
   }, []);
+  return _rows.map(({id, cells}) => {
+    for(const colName of notImplemented) {
+      const columnIndex = columnIds.indexOf(colName);
+        cells[columnIndex] = {
+          cell: {
+            rowId: id,
+            colId: colName
+          },
+          state: 'notImplemented'
+        }
+    }
+    return {id, cells};
+  });
 }
 
-export function makeMatrix({columns = [], tests = [], ...suite}) {
+/**
+ * Makes a matrix from a report.
+ *
+ * @param {object} options - Options to use.
+ * @param {Array<string>} options.columns - Names of implementations tested.
```

Comment:
Should this be renamed to sth like  `implemented` or sth? Just to be consistent with the naming for the other param `notImplemented` which is for implementations that are not tested.

Reply:
You have a good point, but I'm not sure we want to make breaking changes at the 11th hour so maybe we should add a FIXME and make an issue and then update other test suites that depend on this reporter at another time. Another option is to stick with columns and then just make any tests with out a test suite run against it as `notImplemented`. I guess what I'm trying to say is I wrote all of this Sunday afternoon to make a deadline and it could be re-done in the future. At the very least changing this from `columns` to `implemented` would break other test suites that already run weekly.

Reply:
Yeah makes sense, this is doesn't need to be updated now but will have to consider renaming it in future though. Marking this as resolved.